TradeDis/frontend
# fix: search feature bug

**What was done?**
Task: TRAD-109 "Search feature bug"
- Previously, when text was inputted into the home feed search bar, the "My Postings" section would have a weird behaviour and posts would be added to it
- Now, the search bar filters only the trending posts and there is no weird behaviour with the "My Postings" section

**Areas which need special reviewing**
- Currently, only the Trending posts are being filtered, in the future we may want to have the search bar search all posts (my postings, trending and my inquiries)

**Testing**
This test ensures that the search is only filtering the trending posts.
1. Run the frontend on a simulator (`expo start` in the frontend project)
2. Sign in and navigate to the home feed
3. Type in text in the search bar input and ensure that only the posts in "Trending" are being filtered. The posts in "My Postings" and "My Inquiries" should remain the same.
